src/Tribe/Main.php

```diff
@@ -4355,6 +4359,7 @@ public function prepare_to_fix_tagcloud_links() {
 		 * @return string
 		 */
 		public function add_post_type_to_edit_term_link( $link, $term_id, $taxonomy, $context ) {
+			_deprecated_function( __METHOD__, '4.1.2' );
```

Comment:
Optionally throw in the __CLASS__ here too (if you also do it in `prepare_to_fix_tagcloud_links()` that is).